Sources/LeafKit/SendableBox.swift

```diff
@@ -0,0 +1,24 @@
+import NIOConcurrencyHelpers
+
+/// Uses a locking mechanism to ensure Sendability.
+internal final class SendableBox<Value>: @unchecked Sendable {
```

Comment:
Why are we not using the NIO helpers? This doesn't make un-Sendable objects Sendable unfortunately as I've learn't the hard way

Reply:
What types are you thinking of? I don't think we can use types like NIOLoopBound since it requires an eventloop

Reply:
At least not everywhere. Like in global vars.

Reply:
I forgot to add a `Value: Sendable` constraint. I assume that solves most of the problems?

Reply:
Use `NIOLockedValueBox` then rather than rolling our own. This will cause issues with `userInfo` as I'm sure you'll see

Reply:
ah ... the usual problem of outdated comments.
yeah i didn't remember that type exists.